_pages/en/it-picture-medium-term.md

```diff
@@ -6,11 +6,34 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## Citizen
 
+- Mix of old and new..
+- Current portals and accounts
+- OneGC
+- API Store
```

Comment:
Does "the API store" mean the services that are created using government open data/systems? Or is the API store for businesses to use and create add-on services? Just wanted to be sure, if this should be under Citizen header or businesses.

Reply:
https://api.canada.ca/en/homepage :D